mora/service/orgunit.py

```diff
@@ -63,23 +100,26 @@ def get_one_orgunit(c, unitid, unit=None,
     attrs = unit['attributter']['organisationenhedegenskaber'][0]
     rels = unit['relationer']
 
-    r = {
+    r = {}
+    base_info = {
         'name': attrs['enhedsnavn'],
         'user_key': attrs['brugervendtnoegle'],
         'uuid': unitid,
     }
 
-    if validity is not None:
-        r[keys.VALIDITY] = validity
-
     if details is UnitDetails.MINIMAL:
-        pass
+        r.update(base_info)
+
     elif details is UnitDetails.NCHILDREN:
+        r.update(base_info)
+
         children = c.organisationenhed(overordnet=unitid, gyldighed='Aktiv')
 
         r['child_count'] = len(children)
 
-    else:
+    elif details is UnitDetails.FULL:
+        r[keys.ORG_UNIT] = base_info
```

Comment:
It needs to be clear that these are properties of the object itself, and not relations to other objects.

Reply:
Having the objects properties nested one level down is also inconsistent with how every other detail object is structured.

Reply:
Really? It's an explicit request from Anders…

Reply:
(And I agree about your first point — I just wasn't sure whether it affected other objects as well.)

Reply:
Agreed.